src/components/Wiki/History/NoHistoryView.tsx

```diff
@@ -0,0 +1,44 @@
+import { LinkButton } from '@/components/Elements'
+import { Flex, Box, Image, Text } from '@chakra-ui/react'
+import React from 'react'
+import { Wiki } from '@/types/Wiki'
+
+export const NoHistoryView = ({ wiki }: NoHistoryViewProps) => {
+  return (
+    <Flex
+      w={{ base: '100%' }}
+      alignItems={{ base: 'center' }}
+      justifyContent={{ base: 'center' }}
+      direction={{ base: 'column' }}
+      mt={{ base: '22' }}
+      mb={{ base: '96' }}
+    >
+      <Flex
+        w={{ lg: '45%', base: '100%' }}
+        alignItems={{ base: 'center' }}
+        justifyContent={{ base: 'center' }}
+      >
+        <Image
+          flex={1}
+          marginInlineStart="0 !important"
+          src="/images/noWikiHistory.png"
+          w={{ base: '100%', sm: '80%', md: '60%', lg: '15%' }}
+        />
+      </Flex>
+
+      <Box mt={{ lg: '32', base: '12' }} textAlign="center">
```

Comment:
```suggestion
      <Box mt={{ base: '12', lg: '24' }} textAlign="center">
```
Made this 24 as well to balance the margin top at the picture